src/compute/src/server.rs

```diff
@@ -382,47 +396,51 @@ pub async fn compute_node_serve(
 
 /// Check whether the compute node has enough memory to perform computing tasks. Apart from storage,
 /// it is recommended to reserve at least `MIN_COMPUTE_MEMORY_MB` for computing and
-/// `SYSTEM_RESERVED_MEMORY_MB` for other system usage. If the requirement is not met, we will print
-/// out a warning and enforce the memory used for computing tasks as `MIN_COMPUTE_MEMORY_MB`.
+/// `SYSTEM_RESERVED_MEMORY_PROPORTION` of total memory for other system usage. If the requirement
+/// is not met, we will print out a warning and enforce the memory used for computing tasks as
+/// `MIN_COMPUTE_MEMORY_MB`.
 fn validate_compute_node_memory_config(
     cn_total_memory_bytes: usize,
+    reserved_memory_bytes: usize,
     storage_memory_bytes: usize,
 ) -> usize {
     if storage_memory_bytes > cn_total_memory_bytes {
         tracing::warn!(
-            "The storage memory exceeds the total compute node memory:\nTotal compute node memory: {}\nStorage memory: {}\nWe recommend that at least 4 GiB memory should be reserved for RisingWave. Please increase the total compute node memory or decrease the storage memory in configurations.",
+            "The storage memory exceeds the total compute node memory:\nTotal compute node memory: {}\nStorage memory: {}\nReserved memory: {}\nWe recommend that at least 4 GiB memory should be reserved for RisingWave. Please increase the total compute node memory or decrease the storage memory in configurations.",
```

Comment:
Not sure whether we should make our users aware of the "reserved memory" in this warning message. Maybe we should enable users to query out the detailed memory settings via some commands, and just print out some general instructive messages here.

Reply:
OKK, will remove the `reserved memory` log part.